db/data/20210721181311_create_test_experiment.rb

```diff
@@ -0,0 +1,31 @@
+class CreateTestExperiment < ActiveRecord::Migration[5.2]
+  def up
+    if CountryConfig.current_country?("India")
+      experiment = Experimentation::Experiment.create!(name: "production test", experiment_type: "current_patients", state: "new")
+      control_group = experiment.treatment_groups.create!(description: "control")
+      treatment_group = experiment.treatment_groups.create!(description: "professional_request")
+      treatment_group.reminder_templates.create!(message: "notifications.set01.professional_request", remind_on_in_days: -1)
+      treatment_group.reminder_templates.create!(message: "notifications.set02.professional_request", remind_on_in_days: 0)
+      treatment_group.reminder_templates.create!(message: "notifications.set03.professional_request", remind_on_in_days: 1)
+
+      now = Time.current
+      user = User.last # we should find a user we can use or create a new one
+      patient = Patient.create!(id: SecureRandom.uuid, full_name: "Hari AB Tester", status: "active", device_created_at: now, device_updated_at: now, registration_user: user)
```

Comment:
I'm not sure why some of our models don't automatically create uuids on save. I should look into it.

Reply:
Huh, I didn't realize that was the case - I bet its because patients are often created on devices where they will already have a uuid created there.

Reply:
Yup precisely. I don't know if we explored allowing the server to assign the UUID automatically if missing, though. Looks like we've taken the extreme approach of not even touching the thing.